relayer/metrics/metricsService.go

```diff
@@ -171,15 +177,15 @@ func (m *MetricService) storeChainIdData(projectData map[string]map[string]*Aggr
 	}
 }
 
-func (m *MetricService) storeApiTypeData(chainIdData map[string]*AggregatedMetric, data RelayMetrics, successCount int64) {
+func (m *MetricService) storeApiTypeData(chainIdData map[string]*AggregatedMetric, data RelayMetrics, successCount int64, successLatencyValue int64) {
 	apiTypesData, exists := chainIdData[data.APIType]
 	if exists {
-		apiTypesData.TotalLatency += data.Latency
+		apiTypesData.TotalLatency += successLatencyValue
```

Comment:
we are scared of overflow here. 
calculating average could be done differently by taking the current average with weight of successrelays and adding the latest relay latency with weight 1  
use this algorithm instead 

# initialize variables
n = 0  # number of samples seen so far
avg_latency = 0.0  # current average latency

# for each new latency value
n += 1
delta = latency - avg_latency  # compute difference from previous average
avg_latency += delta / n  # update the average latency

# the final average latency is stored in avg_latency

Reply:
why do we need to calculate the avg everytime we have an insert that doesn't seem neccessary

Reply:
@ranlavanet this is a formula for approximate an average and not for the actual average. running the number through excel gives wrong values.

We are not going to overflow the int64, 
we have a limit of maxCU per epoch, which limit the amount of relays
 and we are storing totals for 5min which is less than an epoch.
 
 we will change the type to uint instead int so we will have increased range.